src/fal/cli.py

```diff
@@ -10,93 +12,103 @@
 from faldbt.project import FalDbt, FalGeneralException, FalProject
 
 
-@click.group()
-@click.version_option()
+class FalCli(object):
+
+    def __init__(self, argv):
+        parser = argparse.ArgumentParser(
+            description="Run Python scripts on dbt models",
+            usage='''fal run [<args>]''')
+
+        # Handle version checking
+        version = pkg_resources.get_distribution('fal').version
+        parser.add_argument("-v",
+                            "--version",
+                            action="version",
+                            version=f"fal {version}",
+                            help="show fal version")
+
+        # Handle commands
+        parser.add_argument("command", help="Subcommand to run")
```

Comment:
Shouldn't this be used instead? https://docs.python.org/3/library/argparse.html#sub-commands

The current solution seems a little _hacky_

Reply:
Yeah, I looked at that, and decided to try this way in order to make it easier to incorporate future commands. 

We can define another method in this class and it will automatically turn into a working command.